app/src/main-process/main.ts

```diff
@@ -46,10 +43,10 @@ if (__WIN32__ && process.argv.length > 1) {
   if (handleSquirrelEvent(process.argv[1])) {
     app.quit()
   } else {
-    const url = parseURL(process.argv[1])
-    if (url.name === 'open-repository') {
```

Comment:
@shiftkey I noticed we were only handling the one URL action here. Presumably we should handle it the same as we do everywhere else?